guides-and-tutorials/running-a-signer/best-practices-to-run-a-signer.md

```diff
@@ -57,7 +48,7 @@ without interruption.
 
 ### Upgrade procedures
 
-- Upgrading one Signer instance (of those running redundantly) at the time.
+- Upgrading one Signer instance at the time.
```

Comment:
What does that mean? Maybe 
"During upgrade, your signer is offline and won't sign any blocks. Ensure that the downtime is as short as possible."